content/metadata.yaml

```
@@ -52,7 +73,12 @@ authors:
     X: TODO
     email: david@ccdatalab.org
     contributions:
-      - TODO
+      - Methodology
+      - Software
+      - Validation
+      - Data curation
+      - Writing - Review & Editing
+      - Visualization
```

I don't interpret Visualization to include display in the Portal since it says (emphasis mine):

> Preparation, creation and/or presentation of the published work, **specifically
visualization/ data presentation**

```suggestion
      - Resources
```

But I think including provision of computing resources makes sense